docs/api-guide.md

```diff
@@ -216,177 +231,191 @@ Set<String> namespaces = admin.getNamespaceNames();
 
-This section explains how to execute transactional operations with Transactional API in ScalarDB.
+This section explains how to execute transactional operations by using the Transactional API in ScalarDB.
 
-### Get a DistributedTransactionManager instance
+### Get a `DistributedTransactionManager` instance
 
-You need to get a `DistributedTransactionManager` instance to execute transactional operations.
-You can get it in the following way:
+You first need to get a `DistributedTransactionManager` instance to execute transactional operations.
+
+To get a `DistributedTransactionManager` instance, you can use `TransactionFactory` as follows:
 
 ```java
-TransactionFactory transactionFactory = TransactionFactory.create("<configuration file path>");
+TransactionFactory transactionFactory = TransactionFactory.create("<CONFIGURATION_FILE_PATH>");
 DistributedTransactionManager transactionManager = transactionFactory.getTransactionManager();
 ```
 
-Once you have executed all transactional operations, you should close the `DistributedTransactionManager` instance as follows:
+After you have executed all transactional operations, you should close the `DistributedTransactionManager` instance as follows:
 
 ```java
 transactionManager.close();
 ```
 
-### Begin/Start a transaction
+### Begin or start a transaction
+
+Before executing transactional CRUD operations, you need to begin or start a transaction.
 
-You need to begin/start a transaction before executing transactional CRUD operations.
-You can begin/start a transaction as follows:
+You can begin a transaction as follows:
 
 ```java
-// Begin a transaction
+// Begin a transaction.
 DistributedTransaction transaction = transactionManager.begin();
+```
 
-Or
+Or, you can start a transaction as follows:
 
-// Start a transaction
+```java
+// Start a transaction.
 DistributedTransaction transaction = transactionManager.start();
 ```
 
-You can also begin/start a transaction with specifying a transaction ID as follows:
+Alternatively, you can use the `begin` method for a transaction by specifying a transaction ID as follows:
 
 ```java
-// Begin a transaction with specifying a transaction ID
-DistributedTransaction transaction = transactionManager.begin("<transaction ID>");
+// Begin a transaction with specifying a transaction ID.
+DistributedTransaction transaction = transactionManager.begin("<TRANSACTION_ID>");
+```
 
-Or
+Or, you can use the `start` method for a transaction by specifying a transaction ID as follows:
 
-// Start a transaction with specifying a transaction ID
-DistributedTransaction transaction = transactionManager.start("<transaction ID>");
+```java
+// Start a transaction with specifying a transaction ID.
+DistributedTransaction transaction = transactionManager.start("<TRANSACTION_ID>");
 ```
 
-Note that you must guarantee uniqueness of the transaction ID in this case.
+{% capture notice--info %}
+**Note**
+
+You must guarantee the uniqueness of the transaction ID in this case.
```

Comment:
Should we mention how to get the transaction ID like we do in [`two-phase-commmit-transactions.md` (commit in PR #1082)](https://github.com/scalar-labs/scalardb/blob/f4360781940685d5c9bc86daa425fca870ae7f2f/docs/two-phase-commit-transactions.md?plain=1#L93)?

````suggestion
You must guarantee the uniqueness of the transaction ID in this case. To get the transaction ID with `getId()`, you can specify the following:

```java
tx.getId();
```
````

Reply:
If the passed transaction ID is unique, it probably works properly regardless of whether it was obtained by calling `getID` on the existing transaction. For example, passing a randomly generated UUID would be fine as well.

But, I think it would not be a good document for users since there are few use cases with such a randomly generated ID. Also, uniqueness is an internal matter and wouldn't involve users.

IMO, we should note first what is a use-case for passing a transaction ID when starting a new transaction, then explain how to do it as we do in the section for `join` or `resume`

If there isn't a particular use case, we may delete the section.

Reply:
Thank you for the explanation. I think we have some other documents that include a note similar to this one.

@brfrn169 Are there any particular use cases that we should mention? Or are you OK with deleting this note? If we should delete this note, should we remove it from the other documents too?

Reply:
@josh-wong We should discuss the `getId()` matter in the `Join a transaction` section:
https://github.com/scalar-labs/scalardb/pull/1084/files#diff-d2b4776d2d9b9950fc1f58e9dd0e38550a526976d80e2232f4cb2dcfb84171b2R359

It might also be worth mentioning in the `Resume a transaction` section:
https://github.com/scalar-labs/scalardb/pull/1084/files#diff-d2b4776d2d9b9950fc1f58e9dd0e38550a526976d80e2232f4cb2dcfb84171b2R370

And, what @feeblefakie suggested should be included in this `Begin or start a transaction` section.

Reply:
Got it. I've added the note about `getId()` to the `Join a transaction` and `Resume a transaction` sections, and added the part that @feeblefakie suggested to this section (`Begin or start a transaction`).